CHANGELOG.md

```diff
@@ -13,6 +13,13 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 ### :bug: (Bug Fix)
 
+* fix(sdk-metrics): use default Resource to comply with semantic conventions [#3410](https://github.com/open-telemetry/opentelemetry-js/pull/3410) @pichlermarc
```

Comment:
```suggestion
* fix(sdk-metrics): use default Resource to comply with semantic conventions [#3410](https://github.com/open-telemetry/opentelemetry-js/issues/3410) @pichlermarc
```

Reply:
Why use the issue number and the pull number?

Reply:
This is supposed to be linked to this PR.

Reply:
Must've copied from the wrong tab :sweat:. Fixed in [53070db](https://github.com/open-telemetry/opentelemetry-js/pull/3411/commits/53070dbdd067f5fee0cd88462c41a9fa4ad3ba1e) :slightly_smiling_face: